shotover-proxy/tests/test-configs/cassandra/cluster-v5/topology-encode.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
this file is unused

Reply:
I had intended to use them in a followup, but I'll remove from this PR.